app/views/shared/_nav_bar.html.erb

```diff
@@ -0,0 +1,109 @@
+<nav class="<%= style_class_for_environment %>">
```

Comment:
Nitpick, can we use 2-space indents instead of 4-space please? I'm happy to rejigger this automatically if you'd like. Lemme know 😄

Reply:
Yes sorry! Daniel's text editor is likely using 4-space indents. I'll make sure all files use 2-space indents 👌

Reply:
Yes sorry! I'll make sure all files use 2-space indents 👌